connectors/src/api/get_resources_titles.ts

```diff
@@ -15,12 +16,14 @@ type GetResourcesTitlesRequestBody = t.TypeOf<
   typeof GetResourcesTitlesRequestBodySchema
 >;
 
-type GetResourcesTitlesResponseBody = {
-  resources: {
-    internalId: string;
-    title: string | null;
-  }[];
-};
+type GetResourcesTitlesResponseBody =
```

Comment:
We could create a wrapper something like `WithConnectorsAPIErrorResponse<T>` so we can use it everywhere.

Reply:
(done) in last commit